src/components/ProductList.jsx

```
@@ -19,14 +19,54 @@ function ProductList() {
+    }
+    if (res.ok) {
+      fetchProducts();
+    } else {
+      console.error("POST response error");
+    }
   };
 
   const handleEdit = async (updatedProduct) => {
     // TODO: PUT API를 호출하고 fetchProducts() 호출
+    try {
+      const res = await fetch(`${API_URL}/${updatedProduct.id}`, {
```

ESLint

[error] 'res' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)